repository/src/main/resources/db/changelog/db.changelog-rdm-3940.xml

```diff
@@ -0,0 +1,13 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="rdm-3940" author="rafal.kalita@hmcts.net">
+        <modifyDataType columnName="label"
+                        newDataType="text"
```

Comment:
in other places where we define a label we are using a varchar with a max length specified. I assume that's to prevent very long labels that would cause issues on UI? Shall we do the same here?

Reply:
ok, will change to varchar(200)

Reply:
It is worth noticing that this label is an override, so I am dropping the NOT_NULL constraint. Tested locally and it will use the original label if no label provided.